utils/swift_build_support/swift_build_support/products/llvm.py

```diff
@@ -351,6 +352,36 @@ def build(self, host_target):
             llvm_cmake_options.define('LLVM_INCLUDE_TESTS', 'NO')
             llvm_cmake_options.define('CLANG_INCLUDE_TESTS', 'NO')
 
+        build_root = os.path.dirname(self.build_dir)
```

Comment:
We will want them in the generated toolchains.

Reply:
I'm slightly torn about this; I had intended to get rid of the `.cfg` files by patching the compiler drivers, particularly now I've changed the triple to a special one for the static SDK (`<arch>-swift-linux-musl` instead of `<arch>-unknown-linux-musl`). I think though that we probably do want to add them in the install phase for now; we can always remove them again later if it's a problem.